src/mapgen_functions.cpp

```diff
@@ -3542,7 +3542,7 @@ void mapgen_s_gun(map *m, oter_id terrain_type, mapgendata dat, int, float densi
         for (int i = 0; i < SEEX * 2; i++) {
             for (int j = 0; j < SEEY * 2; j++) {
                 if ((i == 2 || i == SEEX * 2 - 3) && j > 6 && j < SEEY * 2 - 1) {
-                    m->ter_set(i, j, t_wall);
+                    m->ter_set(i, j, t_brick_wall);
```

Comment:
I did that for consistency's sake. Everything else from terrain is in there, so when I added brick_wall, I added halfway.